site/content/en/docs/Advanced/scheduling-and-autoscaling.md

```diff
@@ -97,6 +97,12 @@ This affects the Cluster autoscaler, Allocation Scheduling, Pod Scheduling and F
 
 #### Cluster Autoscaler
 
+{{< alert title="SafeToEvict Feature Gate" color="info" >}}
+The [Alpha]({{< ref "/docs/Guides/feature-stages.md#alpha" >}}) `SafeToEvict` feature allows
```

Comment:
I'm not sure how to best phrase this, but we want to convey that it provides a better way to control disruption than the annotation (which is only for the autoscaler) as it allows the game to establish a contract about it's behavior. 

Maybe you cover this more in the longer doc - reviewing this small change first.

Reply:
I tried to phrase it a little better. WDYT about actually just providing an `eviction` sample here? i.e.  split the samples up with 

##### Using the `eviction` API
<blah>

##### The hard way
<blah>

or something?